Apps/W1/Shopify/app/src/Base/Pages/ShpfyShopCard.Page.al

```diff
@@ -974,6 +998,25 @@ page 30101 "Shpfy Shop Card"
                         Report.Run(Report::"Shpfy Sync Shipm. to Shopify");
                     end;
                 }
+                action(SyncPostedSalesInvoices)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Sync Posted Sales Invoices';
+                    Image = Export;
+                    Promoted = true;
+                    PromotedCategory = Category5;
+                    PromotedIsBig = true;
+                    PromotedOnly = true;
+                    ToolTip = 'Synchronize posted sales invoices to Shopify. The action can be used only if the Posted Invoice Sync field is enabled in the Shopify shop.';
```

Comment:
Maybe modify the tooltip to say that the sync will only be performed if the setting is enabled? Since now the action can always be used?